wiki/People/The_Team/Beatmap_Nominators/Rules/en.md

```diff
@@ -36,7 +36,7 @@ Each mode has its own required minimum for monthly activity.
 ## Nomination
 
 - **Do not nominate beatmaps with unrankable issues.** The General Ranking Criteria and additionally all mode-specific Ranking Criteria apply to all sets with mode-specific difficulties.
-- **Do not nominate beatmapsets that you contributed to.** This applies to your own beatmaps, collaboration difficulties you participate in as well as beatmaps that you hitsounded or made a storyboard for.
+- **Do not nominate beatmapsets that you contributed to.** This applies to your own beatmaps, collaboration difficulties you participate in as well as beatmaps you made other major contributions to in the form of hitsounds, slider velocity changes (osu!taiko & osu!mania) and storyboards.
```

Comment:
```suggestion
- **Do not nominate beatmapsets that you contributed to.** This includes any major contribution to mapping, hitsounding, storyboarding, skinning, or slider velocity editing.
```

Reply:
```suggestion
- **Do not nominate beatmapsets that you contributed to.** This includes any major contribution such as mapping, hitsounding, storyboarding, skinning, or slider velocity editing.
```

Reply:
why not just say "this includes ANY CONTRIBUTION at all, no matter how small" and call it a day?

Reply:
that would possibly result in long and pointless discussions of "what even contribution is" nature, newer people could also be confused with it if they don't know how modding systems and stuff works. being more straightforward makes things simpler and more clear i suppose

Reply:
i'm not sure how this isn't already covered in the initial wording

if SV is as impactful as the OP says to taiko then it is already "contribution" at a base line level without needing to be explicitly stated in the ranking criteria

the fact this PR exists is indicative of shenanigans going on somewhere that needs resolution, probably not the RC itself

Reply:
> Ephemeral: i'm not sure how this isn't already covered in the initial wording

it is, this PR is just clarifying that because the rule didn't do a good job of conveying that its examples were just examples. I removed the "rule change" tag and will apply Noffy's suggestion

> peppy: why not just say "this includes ANY CONTRIBUTION at all, no matter how small" and call it a day?

I think this would be a rule change because currently BNs can make small contributions and still nominate-- and arguably modding itself is a small contribution, depending on how it's done. it could also be a source of confusion like Venix said